lesson11/js/main.js

```diff
@@ -0,0 +1,67 @@
+"use strict";
+const menuContainer = document.getElementById("js-menu_container");
+const menuLists = document.createElement("ul");
+const loading = document.getElementById("js-loading");
+
+function showLoadingImg() {
+  const loadingImg = document.createElement("img");
+  loadingImg.src = "images/loading-circle.gif";
+  loadingImg.classList.add("el_loadingImg");
+
+  loading.appendChild(loadingImg);
+}
+
+function createMenuList({data}) {
+  const fragment = document.createDocumentFragment();
+
+  data.forEach((value) => {
+      const listItem = document.createElement("li");
+  
+      const listImg = document.createElement("img");
+      listImg.src = value.img;
+      listImg.alt = value.alt;
+      listImg.width = 40;
+      listImg.height = 40;
+  
+      const listAnchor = document.createElement("a");
+      listAnchor.href = value.a;
+      listAnchor.textContent = value.text;
+  
+      fragment.appendChild(listItem).appendChild(listAnchor).appendChild(listImg);
+
+    });
+
+  menuContainer.appendChild(menuLists).appendChild(fragment);
+}
+
+async function fetchMenuContents(){
+  showLoadingImg();
+
+  const menuDataURL = 'https://jsondata.okiba.me/v1/json/Aticp210816214718';
+  const response = await fetch(menuDataURL);
+  
+  if(response.status === 200){
+    return await response.json();
+
+  }else{
+   console.error("Could not get the value");
+  }
+
+}
+
+async function callMenuContents() {
+  try {
+    const menuContents = await fetchMenuContents();
+    createMenuList(menuContents);
+
```

Comment:
I didn't want the line breaks in each of these processes.
The line break is a single line, and the line break between functions is also a single line, so I felt that it could make it difficult to see where the break is.

If the process is so complicated that it's hard to see, or if you want to pause for a while, you may want to add a new line.
Otherwise, I think it's fine without line breaks!

Reply:
Thank you for review!
I removed the line breaks.